app/domain/operations/eligible/build_grant.rb

```diff
@@ -0,0 +1,51 @@
+# frozen_string_literal: true
+
+require "dry/monads"
+require "dry/monads/do"
+
+module Operations
+  module Eligible
+    # Operation to support eligibility creation
```

Comment:
Would be super helpful to have more detail in comment. For example, perhaps some examples for grant key/vals, unless we have that info on the model already.